### PR TITLE
Convert enchannel-zmq-backend specs to jest

### LINKED ISSUE #1892: Not all package tests are run on CI
While going through #1891 I discovered that some tests of recently brought in packages aren't running on CI.
We should add a script to run them or move them over to `jest` ([`jest-codemods`](https://github.com/skovhus/jest-codemods) are very helpful)

- [x] `ansi-to-react` (done in #1891)
- [x] `enchannel`
- [ ] `enchannel-zmq-backend`
- [x] `fs-observable`

### PR DESCRIPTION
**Pull request by lgeiger**
Now all test properly run on CI. Fixes #1892 

I wasn't able to properly mock jmp/zeromq so a small amount of rebuilding is needed. Luckily the Electron app still works after running the tests without rebuilding (probably because we're not hoisting so it requires zeromq from a different location).

### CODE REVIEW

**Comment by rgbkrk** [comment]
Oh yeah, our old mock setup used mock-require and was doing it with relative paths.

**Comment by rgbkrk** [comment]
I'm not sure what's up with codecov saying CI failed. Does it seem like a fluke and that we can just merge this in?

File: package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "test:coverage": "npm run coverage",
     "test:lint": "npm run lint",
     "test:flow": "npm run flow",
-    "test:packages": "jest",
+    "test:packages":
+      "lerna exec --scope enchannel-zmq-backend -- npm rebuild && jest",
```
Comment (rgbkrk) [inline package.json:39]: At least it's explicit here for now. :smile:

**Comment by lgeiger** [comment]
Reseted the Appveyor cache. This should now be good to go.

For the desktop app, I think it's best to wait for [sinon support in jest-codemods](https://github.com/skovhus/jest-codemods/issues/68), before converting fully moving to jest.

**Comment by lgeiger** [comment]
> Oh yeah, our old mock setup used mock-require and was doing it with relative paths.

Which mock setup are you talking about?
`enchannel-zmq-backend` tests did never run after we brought them into the monorepo and at the `nteract/enchannel-zmq-backend` they just used regular zeromq.

**Comment by rgbkrk** [comment]
Oh interesting, oops.

Within the desktop app's tests: https://github.com/nteract/nteract/blob/master/packages/desktop/test/setup.js#L82-L87

**Comment by lgeiger** [comment]
Oh I see, thanks

**Pull request merged**